apps/tables/src/components/TablesTable.vue

```diff
@@ -8,7 +8,9 @@
     </thead>
     <tr v-for="table in tables" :key="table.id">
       <td>
-        <router-link :to="table.id">{{ table.label }}</router-link>
+        <router-link :to="table.id">{{
+          table.label === undefined || table.id
```

Comment:
`table.label === null or table.label === "" `

Reply:
Ok ok, that works better. 😅

Reply:
euuuu, i mean `(table.label === undefined || table.label === null || table.label === "") ? table.id : table.label `